tests/register.spec.coffee

```diff
@@ -115,15 +125,15 @@ describe 'Device Register:', ->
 						deviceType: 'raspberry-pi'
 						uuid: 'asdf'
 						apiKey: 'asdf'
-					, _.noop
-
-					expect(postSpy).to.have.been.calledOnce
-					expect(postSpy.args[0][0].body.uuid).to.equal('asdf')
-					postSpy.restore()
+					, ->
```

Comment:
Can you assert in this callback and the one from the test case below that an error was not passed to the callback?

```coffee
expect(error).to.not.exist
```